File: 88-merge-sorted-array/merge-sorted-array.java
class Solution {
    public void merge(int[] nums1, int m, int[] nums2, int n) {
        for (int j = 0, i = m; j < n; j++) {
            System.out.print(nums1[i]);
            nums1[i] = nums2[j];
            i++;
        }
        Arrays.sort(nums1);
    }
}

File: 88-merge-sorted-array/merge-sorted-array.cpp
class Solution {
public:
    void merge(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        int idxn = n-1 ,idxm = m-1,r = m+n-1;
        while(idxn>=0)
        {
            if(idxm>=0 && nums1[idxm] > nums2[idxn])
            {
                nums1[r] = nums1[idxm];
                idxm--;
            }
            else
            {
                nums1[r] = nums2[idxn];
                idxn--;
            }
            r--;
        } 
    }
};